### PR TITLE
Added static deploy command in install module section

### PR DESCRIPTION
**Pull request by chandru-199**
## Purpose of this pull request

This pull request (PR) - static deploy command is missing in install modules section in this document. So added that with details.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/ext-best-practices/extension-coding/example-module-adminpage.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/ext-best-practices/extension-coding/example-module-adminpage.md [hunk 1]
```diff
@@ -323,6 +323,7 @@ Now that the module is code-complete, run the following commands to install it:
 1. `bin/magento module:enable MyCompany_ExampleAdminNewPage` - If necessary, run this to enable the disabled module.
 1. `bin/magento setup:upgrade` - This command will properly register the module with Magento.
 1. `bin/magento setup:di:compile` - This command compiles classes used in dependency injections.
+1. `bin/magento setup:static-content:deploy` - This command deploys static view files in production mode. If you are in developer mode refer to [Clean static files cache]({{ page.baseurl }}/frontend-dev-guide/cache_for_frontdevs.html#clean_static_cache) for refresh the static view files.
```
Comment (keharper) [inline src/guides/v2.3/ext-best-practices/extension-coding/example-module-adminpage.md:326]:
```suggestion
1. `bin/magento setup:static-content:deploy` - (Production mode only) This command deploys static view files. If you are in developer mode, refer to [Clean static files cache]({{ page.baseurl }}/frontend-dev-guide/cache_for_frontdevs.html#clean_static_cache) to refresh the static view files.
```

**Comment by chandru-199** [comment]
@keharper : Changes made. Please review.

**Comment by keharper** [comment]
running tests

**Comment by chandru199** [comment]
No labels were added for this PR.

**Comment by chandru-199** [comment]
No labels were added for this PR?

**Comment by keharper** [comment]
Apologies. It's been a long day.

**Comment by chandru-199** [comment]
Thanks @keharper

**Pull request merged**